browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -367,11 +370,6 @@ gfx::Size VerticalTabStripRegionView::GetMinimumSize() const {
 }
 
 void VerticalTabStripRegionView::Layout() {
-  if (size() == last_layout_size_)
-    return;
```

Comment:
@simonhong This makes the problem where tab strip isn't visible on startup, but couldn't figure out exactly which paret caused that.